checklist.bs

```diff
@@ -288,210 +288,209 @@ syntax specified in [[DXP#api-action-get-request]].
 The test target host system should respond with a 404 Not Found and a JSON body containing the error
 code `NoSuchFootprint`, as specified in [[DXP#api-error-responses]].
 
+## Test Case 012: Receive Notification of PCF Update ## {#tc012}
+
+Advisement: This test case will become mandatory test case with the release of version 2.2 of the Technical Specifications.
```

Comment:
Revise / remove the Advisement, as it is not logically relevant anymore (v2.2 is released). Would revise to make reference somewhere that this test case is only mandatory if the tester is testing conformance to V2.2 or higher of the specifications. This applies to all advisements similar to this one...

Reply:
done